File: Arithmetic_ListFactors.cpp
// Arithmetic.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "ListFactors.h"

#include <iostream>

using namespace std;


int main() {
	try {

		cout << "\n    Welcome to our demonstration sequels\n";
		cout << "Hope you enjoy (and follow) the lessons.\n\n";


		unsigned int start = 1, stop = 100;

		/*
		* Factors of a number.
		*/
		ListFactors f_list(48);
		cout << "\n\n" << f_list.doBusiness() << "\n";

	}	catch (exception& e) {
		cout << "\n" << e.what() << "\n";
	}
    return 0;
}

File: Arithmetic_ListFactors.cs
using System;
using System.Collections.Generic;

namespace Arithmetic_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Welcome to our demonstration sequels");
            Console.WriteLine("Hope you enjoy (and follow) the lessons.");
            Console.WriteLine("\r\n");

            /*
            * List Prime Numbers.
             */
            int candidate = 48;
            List<int> answer;

            ListFactors factors = new ListFactors(candidate);
            answer = factors.doBusiness();

            Console.WriteLine("Factors of " + candidate + " are:");
            Console.WriteLine(String.Join(", ", answer));

        }
    }
}
